.github/workflows/add-lang.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         id: cache-po4a
         with:
           path: "~/po4a"
-          key: ${{ runner.os }}-po4a
+          key: ${{ runner.os }}-po4a-0.68
```

Comment:
You could use a hash like in the main repo to make the key unique

Reply:
Uh, I had a look but don't have a clue how that works...

Reply:
The cache hasn't been used for a week, which means that any workflow run should now trigger the creation of a new one. So I could leave the key here as "po4a" and merge the new po4a version in the assets repo now. Does that sound ok @ann0see?

Reply:
Use ${{ hashFiles('file1, 'file2') }}... Will open an issue for that soon.